src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -376,9 +375,17 @@ class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
 
   FUZZER_TEST_DATA_REL_PATH = os.path.join('test_data', 'fuzzing')
 
-  def _setup_fuzzer_and_device(self):
-    """ Build a Fuzzer object based on the QEMU values.
-    Call this only after setup_qemu_values()"""
+  def _setup_device_and_fuzzer(self, qemu_is_running=True):
+    """Build a Device and Fuzzer object based on QEMU's settings."""
+
+    # If QEMU isn't currently running, set that up first.
+    if not qemu_is_running:
+      qemu = QemuProcess()
```

Comment:
can we just create a device.start_qemu() method which does these three things? There seems to be no reason to have callers call these 3 methods here rather than a single one.

also related to above comment, this can be removed completely.